app/Jobs/ApplicationDeploymentJob.php

```
@@ -2388,13 +2534,12 @@ private function build_image()
             // Coolify variables are already included in the secrets from generate_build_env_variables
             // build_secrets is already a string at this point
         } else {
-            // Traditional build args approach
-            $this->environment_variables->filter(function ($key, $value) {
-                return str($key)->startsWith('COOLIFY_');
-            })->each(function ($key, $value) {
+            // Traditional build args approach - generate COOLIFY_ variables locally
+            // Generate COOLIFY_ variables locally for build args
+            $coolify_envs = $this->generate_coolify_env_variables();
```

PHPMD (2.15.0)

The variable $coolify_envs is not named in camelCase. (undefined)

(CamelCaseVariableName)